src/topp/TextExporter.cpp

```diff
@@ -242,7 +242,14 @@ namespace OpenMS
   // stream output operator for a ProteinHit
   SVOutStream& operator<<(SVOutStream& out, const ProteinHit& hit)
   {
-    out << hit.getScore() << hit.getRank() << hit.getAccession()
+    // extract protein name from description
+    String protein_name = hit.getDescription();
+    if (protein_name.find("OS=") != std::string::npos)
```

Comment:
It's a "magic string". I agree, but could not come up with anything better.

If the description is of this particular form, i.e. includes an " OS=" the split is done. Otherwise nothing happens.